liberty-maven-plugin/src/it/setup/test-war/pom.xml

```diff
@@ -41,10 +41,6 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
You don't need the compiler-plugin configuration. It should now be inherit from parent.